ci_framework/roles/edpm_build_images/tasks/run.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Build EDPM hardened uefi image
+  when:
+    - (cifmw_edpm_build_images_hardened_uefi | bool) or (cifmw_edpm_build_images_all | bool)
+  environment:
+    DIB_LOCAL_IMAGE: "{{ cifmw_edpm_build_images_dib_local_image_path }}/{{ cifmw_edpm_build_images_base_image_name }}"
+    DIB_YUM_REPO_CONF: "{{ cifmw_edpm_build_images_dib_yum_repo_conf | join(' ') }}"
+    ELEMENTS_PATH: "{{ cifmw_edpm_build_images_elements | join(':') }}"
+  ansible.builtin.shell: |
+    diskimage-builder images/edpm-hardened-uefi-{{ distro_name }}-{{ distro_version }}-stream.yaml > {{ cifmw_edpm_build_images_artifacts }}/edpm_images/edpm_hardened_uefi_image_build.log
+  args:
+    chdir: "{{ cifmw_edpm_image_builder_repo_path }}"
+
+- name: Build ironic-python-agent image
+  when:
+    - (cifmw_edpm_build_images_ironic_python_agent | bool) or (cifmw_edpm_build_images_all | bool)
+  environment:
+    DIB_LOCAL_IMAGE: "{{ cifmw_edpm_build_images_dib_local_image_path }}/{{ cifmw_edpm_build_images_base_image_name }}"
+    DIB_YUM_REPO_CONF: "{{ cifmw_edpm_build_images_dib_yum_repo_conf | join(' ') }}"
+    ELEMENTS_PATH: "{{ cifmw_edpm_build_images_elements | join(':') }}"
+  ansible.builtin.shell: |
+    diskimage-builder images/ironic-python-agent-{{ distro_name }}-{{ distro_version }}-stream.yaml > {{ cifmw_edpm_build_images_artifacts }}/edpm_images/ironic_python_agent_image_build.log
```

Comment:
same comments:-

1) logs should go in logs directory and not to the artifacts directory
2) Please also redirect stderr to a file.